mne/io/kit/coreg.py

```diff
@@ -133,6 +133,9 @@ def _set_dig_kit(mrk, elp, hsp, eeg):
         Digitizer head shape points, or path to head shape file. If more
         than 10`000 points are in the head shape, they are automatically
         decimated.
+    bad_coils : list | []
```

Comment:
`[]` is a type of list. And we often use list/tuple interchangeably so we can just say
```suggestion
    bad_coils : list
```